app/models/invitation_to_account.rb

```diff
@@ -0,0 +1,6 @@
+class AccountInvitation < ApplicationRecord
+  belongs_to :user
+  belongs_to :account
+
+  enum status: { new: 0, accepted: 1 }
```

Comment:
Don't use integer for enums, it's hard to transform these values on the fly when write pure SQL.

Also, I've got another idea - what if we just add nullable "accepted_at" column? If it's null it's "new", if it's filled in by the time when it got accepted, then it's "accepted"? Makes sense?

Reply:
It would be great if we could have invitation expiration time. In this case a user won't understand what is going on by "accepted_at" when time is running out.

Reply:
@andreybakanovsky a good idea, but let's keep it simple for now. A task can be created though to add "sent_at" field.

Reply:
Ок, let's keep it simple.